<!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <meta name="viewport" content="width=device-width, initial-scale=1"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" crossorigin="anonymous"> <link rel="stylesheet" href="/assets/main.css"> <link rel="canonical" href="/publications/"> <link rel="alternate" type="application/rss+xml" title="Computer Aided Biotechnology Group" href="/feed.xml"> <!-- Begin Jekyll SEO tag v2.7.1 --> <title>Publications | Computer Aided Biotechnology Group</title> <meta name="generator" content="Jekyll v4.2.1" /> <meta property="og:title" content="Publications" /> <meta property="og:locale" content="en_US" /> <meta name="description" content="TBD" /> <meta property="og:description" content="TBD" /> <link rel="canonical" href="/publications/" /> <meta property="og:url" content="/publications/" /> <meta property="og:site_name" content="Computer Aided Biotechnology Group" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="Publications" /> <script type="application/ld+json"> {"headline":"Publications","description":"TBD","url":"/publications/","@type":"WebPage","@context":"https://schema.org"}</script> <!-- End Jekyll SEO tag --> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.1/font/bootstrap-icons.css" </head> <body> <header id="nav-header" class="border-dark border-left-0 border-right-0"> <div id="nav-container" class="container position-relative d-flex justify-content-between align-items-center"> <a class="h4 m-0 text-dark" href="/">Computer Aided Biotechnology Group</a> <nav id="nav-menu-container" class="text-right navbar-light"> <input type="checkbox" id="nav-trigger" class="d-none"/> <label for="nav-trigger" aria-label="button" class="m-2 d-inline-block d-md-none navbar-toggler-icon"> </label> <div id="nav-menu" class="d-none d-md-block"> <a class="text-dark ml-3 ml-md-0 p-2 d-block d-md-inline-block" href="/research/">Research</a> <a class="text-dark ml-3 ml-md-0 p-2 d-block d-md-inline-block" href="/group/">The Team</a> <a class="text-dark ml-3 ml-md-0 p-2 d-block d-md-inline-block" href="/publications/">Publications</a> <a class="text-dark ml-3 ml-md-0 p-2 d-block d-md-inline-block" href="/software/">Software</a> <a class="text-dark ml-3 ml-md-0 p-2 d-block d-md-inline-block" href="/teaching/">Teaching</a> <a class="text-dark ml-3 ml-md-0 p-2 d-block d-md-inline-block" href="/news/">News</a> <a class="text-dark ml-3 ml-md-0 p-2 d-block d-md-inline-block" href="/contact/">Contact us</a> </div> </nav> </div> </header> <main aria-label="Content"> <div id="content-container" class="container"> <article> <header class="pt-4 pb-3"> <h1>Publications</h1> </header> <div id="content"> <h2 id="preprints">Preprints</h2> <ol class="bibliography"><li><span id="Wright2021.01.18.427050">Wright, N. R., Wulff, T., Workman, C. T., Rønnest, N. P. &amp; Sonnenschein, N. Emergence of phenotypically distinct subpopulations is a factor in adaptation of recombinant Saccharomyces cerevisiae under glucose-limited conditions. <i>bioRxiv</i> (2021). doi:10.1101/2021.01.18.427050</span> <div> <span><a href="http://doi.org/10.1101/2021.01.18.427050">doi: 10.1101/2021.01.18.427050</a></span> <span><a href="https://www.biorxiv.org/content/early/2021/01/18/2021.01.18.427050">https://www.biorxiv.org/content/early/2021/01/18/2021.01.18.427050</a></span> </div></li> <li><span id="Lennen634105">Lennen, R. M., Jensen, K., Mohammed, E. T., Malla, S., Börner, R. A., Chekina, K., Özdemir, E., Bonde, I., Koza, A., Maury, J., Pedersen, L. E., Schöning, L. Y., Sonnenschein, N., Palsson, B. O., Sommer, M. O. A., Feist, A. M., Nielsen, A. T. &amp; Herrgård, M. J. Adaptive laboratory evolution reveals general and specific chemical tolerance mechanisms and enhances biochemical production. <i>bioRxiv</i> (2019). doi:10.1101/634105</span> <div> <span><a href="http://doi.org/10.1101/634105">doi: 10.1101/634105</a></span> </div></li></ol> <h2 id="peer-reviewed-publications">Peer reviewed publications</h2> <ol class="bibliography"><li><span id="MOL2021123">Mol, V., Bennett, M., Sánchez, B. J., Lisowska, B. K., Herrgård, M. J., Nielsen, A. T., Leak, D. J. &amp; Sonnenschein, N. Genome-scale metabolic modeling of P. thermoglucosidasius NCIMB 11955 reveals metabolic bottlenecks in anaerobic metabolism. <i>Metabolic Engineering</i> <b>65,</b> 123–134 (2021).</span> <div> <span><a href="http://doi.org/https://doi.org/10.1016/j.ymben.2021.03.002">doi: https://doi.org/10.1016/j.ymben.2021.03.002</a></span> </div></li> <li><span id="Wright2020-vd">Wright, N. R., Wulff, T., Palmqvist, E. A., Jørgensen, T. R., Workman, C. T., Sonnenschein, N., Rønnest, N. P. &amp; Herrgård, M. J. Fluctuations in glucose availability prevent global proteome changes and physiological transition during prolonged chemostat cultivations of Saccharomyces cerevisiae. <i>Biotechnol. Bioeng.</i> <b>117,</b> 2074–2088 (2020).</span> <div> <span><a href="http://doi.org/https://doi.org/10.1002/bit.27353">doi: https://doi.org/10.1002/bit.27353</a></span> </div></li> <li><span id="Li2020-qz">Li, S., Jendresen, C. B., Landberg, J., Pedersen, L. E., Sonnenschein, N., Jensen, S. I. &amp; Nielsen, A. T. Genome-Wide CRISPRi-Based Identification of Targets for Decoupling Growth from Production. <i>ACS Synth. Biol.</i> <b>9,</b> 1030–1040 (2020).</span> <div> <span><a href="http://doi.org/https://doi.org/10.1021/acssynbio.9b00143">doi: https://doi.org/10.1021/acssynbio.9b00143</a></span> </div></li> <li><span id="Lu2019-ef">Lu, H., Li, F., Sánchez Benjamı́n J, Zhu, Z., Li, G., Domenzain, I., Marcišauskas, S., Anton, P. M., Lappa, D., Lieven, C., Beber, M. E., Sonnenschein, N., Kerkhoven, E. J. &amp; Nielsen, J. A consensus S. cerevisiae metabolic model Yeast8 and its ecosystem for comprehensively probing cellular metabolism. <i>Nat. Commun.</i> <b>10,</b> 3586 (2019).</span> <div> <span><a href="http://doi.org/10.1038/s41467-019-11581-3">doi: 10.1038/s41467-019-11581-3</a></span> </div></li> <li><span id="Jensen2019-gg">Jensen, P. R., Matos, M. R. A., Sonnenschein, N. &amp; Meier, S. Combined In-Cell NMR and Simulation Approach to Probe Redox-Dependent Pathway Control. <i>Anal. Chem.</i> <b>91,</b> 5395–5402 (2019).</span> <div> <span><a href="http://doi.org/10.1021/acs.analchem.9b00660">doi: 10.1021/acs.analchem.9b00660</a></span> </div></li> <li><span id="Massaiu2019-ov">Massaiu, I., Pasotti, L., Sonnenschein, N., Rama, E., Cavaletti, M., Magni, P., Calvio, C. &amp; Herrgård, M. J. Integration of enzymatic data in Bacillus subtilis genome-scale metabolic model improves phenotype predictions and enables in silico design of poly-γ-glutamic acid production strains. <i>Microb. Cell Fact.</i> <b>18,</b> 3 (2019).</span> <div> <span><a href="http://doi.org/10.1186/s12934-018-1052-2">doi: 10.1186/s12934-018-1052-2</a></span> </div></li> <li><span id="Lieven2018-lw">Lieven, C., Petersen, L. A. H., Jørgensen, S. B., Gernaey, K. V., Herrgard, M. J. &amp; Sonnenschein, N. A Genome-Scale Metabolic Model for Methylococcus capsulatus (Bath) Suggests Reduced Efficiency Electron Transfer to the Particulate Methane Monooxygenase. <i>Front. Microbiol.</i> <b>9,</b> 2947 (2018).</span> <div> <span><a href="http://doi.org/10.3389/fmicb.2018.02947">doi: 10.3389/fmicb.2018.02947</a></span> </div></li> <li><span id="Jensen2019-iw">Jensen, K., Broeken, V., Lærke Hansen, A. S., Sonnenschein, N. &amp; Herrgård, M. J. OptCouple: Joint simulation of gene knockouts, insertions and medium modifications for prediction of growth-coupled strain designs. <i>Metabolic Engineering Communications</i> e00087 (2019). doi:10.1016/j.mec.2019.e00087</span> <div> <span><a href="http://doi.org/10.1016/j.mec.2019.e00087">doi: 10.1016/j.mec.2019.e00087</a></span> </div></li> <li><span id="Jessop-Fabre2019-ri">Jessop-Fabre, M. M. &amp; Sonnenschein, N. Improving Reproducibility in Synthetic Biology. <i>Front Bioeng Biotechnol</i> <b>7,</b> 18 (2019).</span> <div> <span><a href="http://doi.org/10.1101/329714">doi: 10.1101/329714</a></span> </div></li> <li><span id="Lieven2020-qs">Lieven, C., Beber, M. E., Olivier, B. G., Bergmann, F. T., Ataman, M., Babaei, P., Bartell, J. A., Blank, L. M., Chauhan, S., Correia, K., Diener, C., Dräger, A., Ebert, B. E., Edirisinghe, J. N., Faria, J. P., Feist, A. M., Fengos, G., Fleming, R. M. T., Garcı́a-Jiménez Beatriz, Hatzimanikatis, V., Helvoirt, W. van, Henry, C. S., Hermjakob, H., Herrgård, M. J., Kaafarani, A., Kim, H. U., King, Z., Klamt, S., Klipp, E., Koehorst, J. J., König, M., Lakshmanan, M., Lee, D.-Y., Lee, S. Y., Lee, S., Lewis, N. E., Liu, F., Ma, H., Machado, D., Mahadevan, R., Maia, P., Mardinoglu, A., Medlock, G. L., Monk, J. M., Nielsen, J., Nielsen, L. K., Nogales, J., Nookaew, I., Palsson, B. O., Papin, J. A., Patil, K. R., Poolman, M., Price, N. D., Resendis-Antonio, O., Richelle, A., Rocha, I., Sánchez Benjamı́n J, Schaap, P. J., Malik Sheriff, R. S., Shoaie, S., Sonnenschein, N., Teusink, B., Vilaça, P., Vik, J. O., Wodke, J. A. H., Xavier, J. C., Yuan, Q., Zakhartsev, M. &amp; Zhang, C. MEMOTE for standardized genome-scale metabolic model testing. <i>Nat. Biotechnol.</i> <b>38,</b> 272–276 (2020).</span> <div> <span><a href="http://doi.org/https://doi.org/10.1038/s41587-020-0446-y">doi: https://doi.org/10.1038/s41587-020-0446-y</a></span> </div></li> <li><span id="Lieven2018-rc">Lieven, C., Herrgård, M. J. &amp; Sonnenschein, N. Microbial Methylotrophic Metabolism: Recent Metabolic Modeling Efforts And Their Applications In Industrial Biotechnology. <i>Biotechnol. J.</i> 1800011 (2018). doi:10.1002/biot.201800011</span> <div> <span><a href="http://doi.org/10.1002/biot.201800011">doi: 10.1002/biot.201800011</a></span> </div></li> <li><span id="Cardoso2018-hq">Cardoso, J. G. R., Zeidan, A. A., Jensen, K., Sonnenschein, N., Neves, A. R. &amp; Herrgård, M. J. MARSI: metabolite analogues for rational strain improvement. <i>Bioinformatics</i> (2018). doi:10.1093/bioinformatics/bty108</span> <div> <span><a href="http://doi.org/10.1093/bioinformatics/bty108">doi: 10.1093/bioinformatics/bty108</a></span> </div></li> <li><span id="Hansen2017-fq">Hansen, A. S. L., Lennen, R. M., Sonnenschein, N. &amp; Herrgård, M. J. Systems biology solutions for biochemical production challenges. <i>Curr. Opin. Biotechnol.</i> <b>45,</b> 85–91 (2017).</span> <div> <span><a href="http://doi.org/10.1016/j.copbio.2016.11.018">doi: 10.1016/j.copbio.2016.11.018</a></span> </div></li> <li><span id="Cardoso2018-rf">Cardoso, J. G. R., Jensen, K., Lieven, C., Lærke Hansen, A. S., Galkina, S., Beber, M., Özdemir, E., Herrgård, M. J., Redestig, H. &amp; Sonnenschein, N. Cameo: A Python Library for Computer Aided Metabolic Engineering and Optimization of Cell Factories. <i>ACS Synth. Biol.</i> (2018). doi:10.1021/acssynbio.7b00423</span> <div> <span><a href="http://doi.org/10.1021/acssynbio.7b00423">doi: 10.1021/acssynbio.7b00423</a></span> </div></li> <li><span id="Jensen2017">Jensen, K., Cardoso, J. G. R. &amp; Sonnenschein, N. Optlang: An algebraic modeling language for mathematical optimization. <i>The Journal of Open Source Software</i> <b>2,</b> (2017).</span> <div> <span><a href="http://doi.org/10.21105/joss.00139">doi: 10.21105/joss.00139</a></span> </div></li> <li><span id="Zhang2016-zc">Zhang, J., Sonnenschein, N., Pihl, T. P. B., Pedersen, K. R., Jensen, M. K. &amp; Keasling, J. D. Engineering an NADPH/NADP+ redox biosensor in yeast. <i>ACS Synth. Biol.</i> acssynbio.6b00135 (2016). doi:10.1021/acssynbio.6b00135</span> <div> <span><a href="http://doi.org/10.1021/acssynbio.6b00135">doi: 10.1021/acssynbio.6b00135</a></span> </div></li> <li><span id="Beber2012">Beber, M. E., Fretter, C., Jain, S., Sonnenschein, N., Müller-Hannemann, M. &amp; Hütt, M.-T. Artefacts in statistical analyses of network motifs: general framework and application to metabolic networks. <i>Journal of the Royal Society, Interface / the Royal Society</i> <b>9,</b> 3426–3435 (2012).</span> <div> <span><a href="http://doi.org/10.1098/rsif.2012.0490">doi: 10.1098/rsif.2012.0490</a></span> </div></li> <li><span id="Bergdahl2015">Bergdahl, B., Sonnenschein, N., Machado, D., Herrgård, M. &amp; Forster, J. in <i>Fundamental Bioengineering</i> 143–182 (Wiley-VCH Verlag GmbH &amp; Co. KGaA, 2015). doi:10.1002/9783527697441.ch06</span> <div> <span><a href="http://doi.org/10.1002/9783527697441.ch06">doi: 10.1002/9783527697441.ch06</a></span> </div></li> <li><span id="Bordbar2015">Bordbar, A., McCloskey, D., Zielinski, D. C., Sonnenschein, N., Jamshidi, N. &amp; Palsson, B. Ø. Personalized Whole-Cell Kinetic Models of Metabolism for Discovery in Genomics and Pharmacodynamics. <i>Cell Systems</i> <b>1,</b> 283–292 (2015).</span> <div> <span><a href="http://doi.org/10.1016/j.cels.2015.10.003">doi: 10.1016/j.cels.2015.10.003</a></span> </div></li> <li><span id="Cardoso2015">Cardoso, J. G. R., Andersen, M. R., Herrgard, M. J. &amp; Sonnenschein, N. Analysis of genetic variation and potential applications in genome-scale metabolic modeling. <i>Frontiers in bioengineering and biotechnology</i> <b>3,</b> 13 (2015).</span> <div> <span><a href="http://doi.org/10.3389/fbioe.2015.00013">doi: 10.3389/fbioe.2015.00013</a></span> </div></li> <li><span id="Ebrahim2015">Ebrahim, A., Almaas, E., Bauer, E., Bordbar, A., Burgard, A. P., Chang, R. L., Dräger, A., Famili, I., Feist, A. M., Fleming, R. M., Fong, S. S., Hatzimanikatis, V., Herrgard, M. J., Holder, A., Hucka, M., Hyduke, D., Jamshidi, N., Lee, S. Y., Le Novere, N., Lerman, J. A., Lewis, N. E., Ma, D., Mahadevan, R., Maranas, C., Nagarajan, H., Navid, A., Nielsen, J., Nielsen, L. K., Nogales, J., Noronha, A., Pal, C., Palsson, B. Ø., Papin, J. A., Patil, K. R., Price, N. D., Reed, J. L., Saunders, M., Senger, R. S., Sonnenschein, N., Sun, Y. &amp; Thiele, I. Do genome-scale models need exact solvers or clearer standards? <i>Molecular Systems Biology</i> <b>11,</b> 831–831 (2015).</span> <div> <span><a href="http://doi.org/10.15252/msb.20156157">doi: 10.15252/msb.20156157</a></span> </div></li> <li><span id="Kildegaard2014">Kildegaard, K. R., Hallström, B. M., Blicher, T. H., Sonnenschein, N., Jensen, N. B., Sherstyk, S., Harrison, S. J., Maury, J., Herrgard, M. J., Juncker, A. S., Forster, J., Nielsen, J. &amp; Borodina, I. Evolution reveals a glutathione-dependent mechanism of 3-hydroxypropionic acid tolerance. <i>Metabolic engineering</i> <b>26C,</b> 57–66 (2014).</span> <div> <span><a href="http://doi.org/10.1016/j.ymben.2014.09.004">doi: 10.1016/j.ymben.2014.09.004</a></span> </div></li> <li><span id="King2015">King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E. &amp; Palsson, B. Ø. Escher: A Web Application for Building, Sharing, and Embedding Data-Rich Visualizations of Biological Pathways. <i>PLoS computational biology</i> <b>11,</b> e1004321 (2015).</span> <div> <span><a href="http://doi.org/10.1371/journal.pcbi.1004321">doi: 10.1371/journal.pcbi.1004321</a></span> </div></li> <li><span id="Machado2015">Machado, D., Zhuang, K. H., Sonnenschein, N. &amp; Herrgard, M. J. Editorial: Current Challenges in Modeling Cellular Metabolism. <i>Frontiers in bioengineering and biotechnology</i> <b>3,</b> 107 (2015).</span> <div> <span><a href="http://doi.org/10.3389/fcell.2015.00017">doi: 10.3389/fcell.2015.00017</a></span> </div></li> <li><span id="Sonnenschein2011">Sonnenschein, N., Geertz, M., Muskhelishvili, G. &amp; Hütt, M.-T. Analog regulation of metabolic demand. <i>BMC Systems Biology</i> <b>5,</b> 40 (2011).</span> <div> <span><a href="http://doi.org/10.1186/1752-0509-5-40">doi: 10.1186/1752-0509-5-40</a></span> </div></li> <li><span id="Sonnenschein2012">Sonnenschein, N., Golib Dzib, J. F., Lesne, A., Eilebrecht, S., Boulkroun, S., Zennaro, M.-C., Benecke, A. &amp; Hütt, M.-T. A Network Perspective on Metabolic Inconsistency. <i>BMC systems biology</i> <b>6,</b> 41 (2012).</span> <div> <span><a href="http://doi.org/10.1186/1752-0509-6-41">doi: 10.1186/1752-0509-6-41</a></span> </div></li> <li><span id="Sonnenschein2009">Sonnenschein, N., Hütt, M.-T., Stoyan, H. &amp; Stoyan, D. Ranges of control in the transcriptional regulation of <i>Escherichia coli</i>. <i>BMC systems biology</i> <b>3,</b> 119 (2009).</span> <div> <span><a href="http://doi.org/10.1186/1752-0509-3-119">doi: 10.1186/1752-0509-3-119</a></span> </div></li> <li><span id="Sonnenschein2012a">Sonnenschein, N., Marr, C. &amp; Hütt, M.-T. A topological characterization of medium-dependent essential metabolic reactions. <i>Metabolites</i> <b>2,</b> 632–647 (2012).</span> <div> <span><a href="http://doi.org/10.3390/metabo2030632">doi: 10.3390/metabo2030632</a></span> </div></li> <li><span id="Thiele2013">Thiele, I., Swainston, N., Fleming, R. M. T., Hoppe, A., Sahoo, S., Aurich, M. K., Haraldsdottir, H., Mo, M. L., Rolfsson, O., Stobbe, M. D., Thorleifsson, S. G., Agren, R., Bölling, C., Bordel, S., Chavali, A. K., Dobson, P., Dunn, W. B., Endler, L., Hala, D., Hucka, M., Hull, D., Jameson, D., Jamshidi, N., Jonsson, J. J., Juty, N., Keating, S., Nookaew, I., Le Novère, N., Malys, N., Mazein, A., Papin, J. A., Price, N. D., Selkov, E., Sigurdsson, M. I., Simeonidis, E., Sonnenschein, N., Smallbone, K., Sorokin, A., Beek, J. H. G. M. van, Weichart, D., Goryanin, I., Nielsen, J., Westerhoff, H. V., Kell, D. B., Mendes, P. &amp; Palsson, B. Ø. A community-driven global reconstruction of human metabolism. <i>Nature biotechnology</i> <b>31,</b> 419–425 (2013).</span> <div> <span><a href="http://doi.org/10.1038/nbt.2488">doi: 10.1038/nbt.2488</a></span> </div></li></ol> </div> </article> </div> </main> <footer id="site-footer"> <div class="container"> <div class="h5 pt-4">Computer Aided Biotechnology Group</div> <div class="row"> <div class="col col-lg-3"> <ul class="list-unstyled"> <li class="text-secondary"> Computer Aided Biotechnology Group </li> <li><a href="mailto:niso@dtu.dk">niso@dtu.dk</a></li> </ul> </div> <div class="col-12 col-lg"> <p class="text-secondary text-justify">TBD</p> </div> </div> </div> </footer> </body> </html>
